docs/src/terminology.md

```diff
@@ -128,11 +142,11 @@ See [cross-program invocation](terminology.md#cross-program-invocation).
 
 ## instruction
 
-The smallest unit of a [program](terminology.md#program) that a [client](terminology.md#client) can include in a [transaction](terminology.md#transaction). Within its processing code, an instruction may contain one or more [cross-program invocations](terminology.md#cross-program-invocation).
+The code that a [client](terminology.md#client) can include in a [transaction](terminology.md#transaction). 
 
```

Comment:
As mentioned, this is not really code. 

Please use
`The smallest unit of a [program](terminology.md#program) that a [client](terminology.md#client) can include in a [transaction](terminology.md#transaction). An instruction specifies which program it is calling, which accounts it wants to read or modify, and an additional data field serving as auxiliary input to the program. Within its processing code, an instruction may contain one or more [cross-program invocations](terminology.md#cross-program-invocation).`

Reply:
I've updated this. Should be satisfactory.

Reply:
^